package.json

```diff
@@ -47,7 +47,8 @@
     "update-cli-packages": "./scripts/update-cli-dependencies.sh && yarn",
     "extract-api": "lerna run extract-api",
     "verify-api-extract": "yarn extract-api && ./scripts/verify-extract-api.sh",
-    "verify-yarn-lock": "./scripts/verify-yarn-lock.sh"
+    "verify-yarn-lock": "./scripts/verify-yarn-lock.sh",
+    "update-test-timing-data": "ts-node ./scripts/cci/get-test-timing-from-cci-job-metrics.ts"
```

Comment:
do we have to run this periodically or is it run automatically in a pre-commit hook to split test files?

Reply:
This will be run periodically(per month) as it now fetches under the 30 day window. The data from circle ci is just for temporary usage as an initial data feed. This will be migrated to codebuild as well once the e2e workflow is migrated.